src/main/java/Duke.java

```diff
@@ -1,10 +1,140 @@
+import java.util.Scanner;
+
 public class Duke {
+    static int taskCount = 0;
+    static Task[] taskList = new Task[100];
```

Comment:
the name tasks can be considered for the name of the array to show that the array contains multiple tasks

Reply:
thank you! but i think i will keep it as this for now (: